flink-runtime/src/test/java/org/apache/flink/runtime/jobgraph/ScheduleModeTest.java

```diff
@@ -18,6 +18,8 @@
 
 package org.apache.flink.runtime.jobgraph;
```

Comment:
Move the unit test to api.common package?

Reply:
good point! I'm moving